cordova-common/src/superspawn.js

```diff
@@ -22,7 +22,7 @@ var fs = require('fs');
 var path = require('path');
 var _ = require('underscore');
 var Q = require('q');
-var shell = require('shelljs');
+var which = require('shelljs/src/which');
```

Comment:
Also, does this really save any time? Lots of sources require shelljs, so I'd expect it to be fully loaded somewhere else anyway.